### DISCUSSION TITLE
Filtering out only open issues to be visible in the board

### DISCUSSION THREAD

**Discussion by anadi2311** [community/community#7817] [Projects and Issues]
Really like the board but
It will be great if I can remove closed issues from the table(violet) and see only the open (green). It is getting unnecessarily cluttered with the closed one.
![image](https://user-images.githubusercontent.com/47948789/142138106-b4c2dd03-a46c-4163-9cd3-89907a8c406c.png)

**Comment by sercand** [comment]
Beginning of a project it is not visible but near end, it is really hard to see what is remaining. Hiding the closed issues should be the default. We added a "status" field and using it to filter out completed issues.
